src/components/signMessage/signMessage.js

```diff
@@ -15,7 +15,9 @@ class SignMessage extends React.Component {
   render() {
     const { account, t, history } = this.props;
     const header = t('Sign a message');
-    const message = t('Signing a message with this tool indicates ownership of a privateKey (secret) and provides a level of proof that you are the owner of the key.');
+    const message = t('You can use your passphrase to sign a message. ') +
+      t('This signed message can prove that you are the owner of the account, ') +
+      t('since only your passphrase can produce it. We recommend including date & time or a specific keyword.');
```

Comment:
It's not good to split one sentence into multiple translation strings, because in another language the sentence structure might be different and also the translators will miss the context of where the sentence starts.